pages/glossary/focusable.md

```diff
@@ -13,10 +13,14 @@ An element is _focusable_ if one or both of the following are true:
 - the element is part of [sequential focus navigation][]; or
 - the element has a [tabindex value][] that is not null.
 
-**Exception**: Elements that lose focus immediately after the focus event is fired are not considered _focusable_.
+**Exception**: Elements that lose focus during a period of up to 1 second after gaining focus, without the user interacting with the page the element is on, are not considered _focusable_.
 
-The [tabindex value][] of an element is the value of the [tabindex attribute][] parsed using the [rules for parsing integers](https://html.spec.whatwg.org/#rules-for-parsing-integers). For the [tabindex value][] to be different from null, the [tabindex attribute][] must be a [valid integer](https://html.spec.whatwg.org/#valid-integer).
+Notes:
 
+- The 1 second time span is an arbitrary limit which is not included in WCAG. Given that scripts can manage the focus state of elements, testing the focused state of an element consistently would be impractical without a time limit.
+- The [tabindex value][] of an element is the value of the [tabindex attribute][] parsed using the [rules for parsing integers][]. For the [tabindex value][] to be different from null, it needs to be [parsed][rules for parsing integers] without errors.
```

Comment:
Is this necessary? What does this add over just linking to the definition? Can we do this in a way that doesn't use a note. Notes suggest content that doesn't impact the outcome, it seems like this could?

Reply:
This is taken from the definition (it does not add to the definition). I added it here because people might confuse the tabindex value with the tabindex attribute, and wrongly assume that all that is needed is a non-empty attribute.

Nothing against removing it, if you don't think it adds any value?